src/network/networkd-route.c

```diff
@@ -1375,6 +1376,24 @@ int link_set_routes(Link *link) {
         if (r < 0)
                 return r;
 
+        HASHMAP_FOREACH(route, link->network->routes_by_section) {
+                MultipathRoute *m;
+
+                ORDERED_SET_FOREACH(m, route->multipath_routes) {
+                        if (isempty(m->ifname))
+                                continue;
+
+                        r = resolve_interface(&link->manager->rtnl, m->ifname);
+                        if (r < 0) {
+                                log_link_debug_errno(link, r,
+                                                     "Failed to resolve interface name '%s' for multipath route, "
+                                                     "ignoring the interface name: %m", m->ifname);
```

Comment:
Hmm, shouldn't we instead treat this is a non-existent interface, and drop the route? Actually that's what the code did before, so if we networkd was started late, when all interfaces were already up, it would just reject routes with non-existent intefaces, and now they're accepted.

Reply:
Yeah, but this code will be dropped soon by later commits.

Reply:
OK. Such incomplete routes are now skipped, but as said in the above, this change will be dropped in the later commits.